### ISSUE TITLE
Game crashes during AI's turn

### ISSUE THREAD

**Issue by Laz0rG** [LoneGazebo/Community-Patch-DLL#8539]
_1. Mod version (X.Y.Z, e.g. 1.3.4):_
Version 1.3.4

_2. Mod list (if using Vox Populi only, leave blank):_
infoaddict (v. 22)
Promotions - Level Promotions (v. 3)
Really Advanced Setup (v. 15)
World Congress Reformation (v. 8)
Global - Faith Purchases in Puppets (v. 1)
(1) Community Patch (v. 101)
(2) Community Balance OVerhaul (v.14)
(3) CSD for VP (v.29)
(4) Civ IV Diplomatic Features (v.12)
(5) More luxuries (v. 156)
(over) More Unique COmponents for VP (v. 74)
Unique City States (v. 6)

Also, I compiled my own cvgamecore_expansion2.dll with the only difference being that I disabled pop loss on settlers (not sure if that is relevant)

_3. Error description:_ 
for whatever reason the game will crash upon certain players turns without fail. Problem is rectified sometimes by forcing the AI at hand to declare peace, but in the example of this save, the Aztecs crash the game regardless unless I delete them from the game using IGE.

_4. Steps to reproduce (optional):_
1. load save and appropriate mods
2. try to advance the turns
3. Game crashes during the Aztec's turn
4. delete them and behold as the game does not crash. 

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

[Logs, CVMiniDump, and save file.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/7873734/Logs.CVMiniDump.and.save.file.zip)

**Comment by innocuousm8** [comment]
Had the same thing happen in two different games now. I'm a random incompetent pleb so if there's a fix and a guide in layman I'd be super happeh.  Both games Russia. One game crashes on Carthage the other on Venice. Both games are abouts the information era and within' a couple hundred turns of one another.  Mod list is the same except for world congress reformation, really advanced setup, and faith puppets thingie. I have my own mod that removes xcom, stealth bombers, death robots, nukes.

**Comment by ilteroi** [comment]
try this one and see if it helps. should be savegame compatible ...
[CvGameCore_Expansion2_v1_3_5unofficial_.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/7902875/CvGameCore_Expansion2_v1_3_5unofficial_.zip)

**Comment by Laz0rG** [comment]
> try this one and see if it helps. should be savegame compatible ... [CvGameCore_Expansion2_v1_3_5unofficial_.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/7902875/CvGameCore_Expansion2_v1_3_5unofficial_.zip)

Unfortunately the only result is a crash to desktop when loading the save file, before the leader screen appears.

**Comment by RecursiveVision** [comment]
Think this is probably fixed for next version
